index.html

```diff
@@ -190,5 +189,73 @@ <h3 class="card-discription">Skills</h3>
         </div>
       </section>
     </main>
+    <!-- Contact form Start here -->
+    <footer>
+      <section class="flex-container">
+        <h2 class="flex-item-1">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form
+          action="https://formspree.io/f/xoqrpder"
+          method="POST"
+          class="flex-item-2"
+        >
+          <ul>
+            <li>
+              <input
+                type="text"
+                name="name"
+                required
+                maxlength="30"
+                placeholder="Full Name"
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="email"
+                required
+                placeholder="Email Address"
```

Comment:
- [ Optional ] In my opinion, you could have achieved the same thing without using `<ul> <li>` in here. Besides you code could have been leaner and more readable. e.g. 

```
<form action>
  <input name="fullname"/>
  <input name="email"/>
  <textarea  name="message">
  <button type="submit">
</form>
```